makefiles/blob.inc.mk

```diff
@@ -0,0 +1,34 @@
+#
+# makes any file in BLOBS available via '#include "blob/<filename>.h"
+#
+# Usage:
+#
+# Add this to an application or module Makefile:
+#
+#     BLOBS += foo.ext
+#
+# Then include in C file or header:
+#
+#    #include "blob/foo.ext.h"
+#
+
+# use "blobs/blob" so the headers can be included as "blob/foo.h", but
+# we don't have to add $(BINDIR)/$(MODULE) to the include path.
+BLOB_HDR_DIR ?= $(BINDIR)/$(MODULE)/blobs/blob
+BLOB_H := $(BLOBS:%=$(BLOB_HDR_DIR)/%.h)
+
+ifneq (, $(BLOB_H))
+	# add $(BINDIR)/$(MODULE)/blobs to include path
```

Comment:
Looks like there's a tab instead of 2 spaces here (and the line below)

Reply:
fixed